tests/data/rawvoice_metamark.json

```diff
@@ -0,0 +1,5 @@
+{
+  "title": "Namespace Test",
+  "episodes": [],
+  "metamark": "Tom: Loved how you discuss how to make cookies with a toaster oven."
```

Comment:
There can probably be multiple metamarks, not just one? Also, we probably also need/want all the attributes. As it is implemented right now, it isn't very useful.